graphql-api/src/queries/coverage-queries.ts

```diff
@@ -68,6 +76,28 @@ const fetchCoverage = async (esClient: any, { index, contig, regions, bucketSize
     },
   })
 
+  console.log(
```

Comment:
This should be removed. Probably the easiest way is to edit the commit with an `edit` command during a rebase.